.github/CONTRIBUTING.md

```diff
@@ -327,6 +327,21 @@ This prevents nesting levels from getting deeper then they need to be.
   * WRONG: list(a = "b")
   * RIGHT: list("a" = "b")
 
+#### Bitflags
+* We prefer using bitshift operators instead of directly typing out the value. I.E.
+    ``` 
+    #define MACRO_ONE (1<<0)
+    #define MACRO_TWO (1<<1)
+    #define MACRO_THREE (1<<2)
+    ```
+    Is preferable to
+    ``` 
+    #define MACRO_ONE 0
+    #define MACRO_TWO 1
+    #define MACRO_THREE 2
```

Comment:
These are no equivalent. The first group of defines define 1 2 4, and the second one (as you can see) 0 1 2 .
I guess you meant 
```
    #define MACRO_ONE 1
    #define MACRO_TWO 2
    #define MACRO_THREE 4
```
?

Reply:
Woops. Let me fix that.